docs/working-drafts/issue-29-erratum.md.adoc

```diff
@@ -0,0 +1,135 @@
+// please start a new sentence on a new line.
+// this facilitates commentary on this document using code-review tools.
+
+= Issue 29 - dfdl:prefixLengthKind Simple Type having DFDL Statement Annotations
+
+
+Sections of the DFDL v1.0 Spec which will require updating include:
+
+## Section 9.5 - Evaluation Order for Statement Annotations
+
+An element (or element reference) having lengthKind 'prefixed' can
```

Comment:
"An element (or element reference) ..." is better stated as "A dfdl:element ..."